demos/layout/templates/layout1.html

```diff
@@ -123,12 +123,12 @@ <h4>Ungrouped transaction list</h4>{Report}
         </tbody>
         <tfoot>
           <tr>
-            <th class="right aligned" colspan="6">Closing balance after {count}29{/} records:</th>
+            <th class="right aligned" colspan="6">Closing balance after {foot_count}29{/} records:</th>
             <th>€1,056.91</th>
             <th>€0.00</th>
           </tr>
           <tr>
-            <th class="right aligned" colspan="6">Total for {nominal}Sales:Limerick{/nominal}:</th>
+            <th class="right aligned" colspan="6">Total for {foot_nominal}Sales:Limerick{/}:</th>
```

Comment:
bug: tag defined twice with different content

Reply:
If we are reworking this: Usually Regions are written with a capital letter `{Count}`, whereas Tags are written lowercase `{$something}`. Not mandatory, but good style.

Reply:
both are exactly the same :D

Reply:
yes, technically there is no difference, but in most ATK templates regions are capital, while pure tags (with a $) are written lowercase. As I said, not mandatory, but imo good style.